freeipa/src/main/java/com/sequenceiq/freeipa/converter/stack/CreateFreeIpaRequestToStackConverter.java

```diff
@@ -120,7 +120,11 @@ public Stack convert(CreateFreeIpaRequest source, String accountId, Future<Strin
             stack.setNetwork(networkConverter.convert(source.getNetwork()));
         }
         stack.setTelemetry(telemetryConverter.convert(source.getTelemetry()));
-        stack.setBackup(backupConverter.convert(source.getTelemetry()));
+        if (source.getBackupRequest() != null && isNotEmpty(source.getBackupRequest().getStorageLocation())) {
```

Comment:
I think the environment service also needs logic like this. (we need it in both places)

Reply:
Thanks! I added the logic in EnvironmentApiConverter.java 

```
                .withBackup((request.getBackupRequest() != null && isNotEmpty(request.getBackupRequest().getStorageLocation())) ?
                        backupConverter.convert(request.getBackupRequest()) : backupConverter.convert(request.getTelemetry()))
```

I tested again using old UI and old dpcli
<img width="329" alt="Screen Shot 2021-01-27 at 2 04 31 PM" src="https://user-images.githubusercontent.com/67844271/106047027-b1482680-60a8-11eb-941f-2d1ea4d7348a.png">

So now the environment table has the backup populated as log. 
<img width="299" alt="Screen Shot 2021-01-27 at 2 04 14 PM" src="https://user-images.githubusercontent.com/67844271/106047089-c58c2380-60a8-11eb-9b5e-c5f9690cc591.png">

Confirmed backup in log folder. 
<img width="583" alt="Screen Shot 2021-01-27 at 2 07 14 PM" src="https://user-images.githubusercontent.com/67844271/106047267-008e5700-60a9-11eb-8431-411ca5cbf757.png">